src/Ledger/Utxo.lagda

```diff
@@ -15,7 +15,7 @@ open import Ledger.Prelude hiding (Dec₁)
 
 open import Algebra                        using (CommutativeMonoid)
 open import Data.Integer.Ext               using (posPart; negPart)
-open import Data.Nat                       using (_≤?_; _≤_)
+open import Data.Nat as ℕ                  using (_≤?_)
```

Comment:
When I remove `as ℕ` and change each occurrence of `ℕ.≤` to `≤`, type-checking hangs.  When I include `as ℕ` and use `ℕ.≤` instead of `≤` in the `Computational'-UTXO` proof, it type-checks fairly quickly.